app/views/candidate_interface/personal_statement/interview_preferences/edit.html.erb

```diff
@@ -8,12 +8,32 @@
 
       <h1 class="govuk-heading-xl"><%= t('page_titles.interview_preferences') %></h1>
 
-      <p class="govuk-body">Your interview will usually take place over the course of a day and you will need to attend in person.</p>
-      <p class="govuk-body">Give details of any arrangements you might need. Providers can’t always accomodate your preferences.</p>
+      <p class="govuk-body">
+        Interviews usually take place over the course of a day.
+        You'll need to attend in person.
+      </p>
+
+      <p class="govuk-body">
+        Training providers might not have much flexibility when setting a date
+        and time for interview.
+      </p>
+
+      <p class="govuk-body">
+        If there's a reason why you need some flexibility you can tell us about
+        it here.
+      </p>
+
+      <p class="govuk-body">For example:</p>
+
+      <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-7">
+        <li>you have commitments such as employment or caring responsibilities</li>
+        <li>you'll be travelling a long way to get to the interview</li>
+        <li>you'll need some adjustments because you're disabled</li>
```

Comment:
```suggestion
        <li>you’ll be travelling a long way to get to the interview</li>
        <li>you’ll need some adjustments because you're disabled</li>
```